src/agent/src/namespace.rs

```diff
@@ -82,6 +81,9 @@ impl Namespace {
 
         let ns_path = PathBuf::from(&self.persistent_ns_dir);
         let ns_type = self.ns_type;
+        if ns_type == NamespaceType::PID {
```

Comment:
If the `setup` will do nothing, should return before `fs::create_dir_all`

@lifupan @yyyeerbo can you confirm that  the `setup` should not be called by pid namespaces.

Reply:
good point @liubin . I will wait on @lifupan and @yyyeerbo answers.

Reply:
@liubin  @wainersm  Yes, this setup only applied to the other ns except pidns. Thanks for this patch.